### PR TITLE
remove package lock and use yarn

### PR DESCRIPTION
**Pull request by priley86**

### CODE REVIEW

**Comment by priley86** [comment]
oops needs rebase.. fixing...

**Comment by priley86** [comment]
@mturley @AparnaKarve @ZitaNemeckova ok to merge?

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# using yarn
+package-lock = false
```
Comment (AparnaKarve) [inline .npmrc]: @priley86 Is `yarn.lock = true` implicit here?

Comment (priley86) [reply .npmrc]: @AparnaKarve i'm not sure - maybe you have another suggestion? This was the setting in ui-classic.

Comment (AparnaKarve) [reply .npmrc]: Maybe it is implicit.
Let's go with this setting, since classic-ui also has it.

Comment (himdel) [reply .npmrc]: `yarn.lock` is generated always when you run yarn, and there's no real way to disable it,
but we can just keep `.gitignore`ing it, same as ui-classic.

**Pull request merged**